scripts/check-pristine-state

```diff
@@ -1,58 +0,0 @@
-#!/usr/bin/env node
-/**
- * Backpack for Android - Skyscanner's Design System
- *
- * Copyright 2018-2021 Skyscanner Ltd
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-const { promisify } = require('util');
-const { exec } = require('child_process');
-
-const execAsync = promisify(exec);
-
-const command = "git status --porcelain | awk '{print $2}'";
-const [...filesToIgnore] = process.argv;
-
-const exists = item => !!item;
-
-const isNotInList = list => item => list.indexOf(item) === -1;
-
-const init = async () => {
-  const { stdout } = await execAsync(command, {
-    cwd: `${__dirname}/../`,
-  });
-  const changedFiles = stdout
-    .trim()
-    .split('\n')
-    .filter(isNotInList(filesToIgnore))
-    .filter(exists);
-
-  if (changedFiles.length) {
-    const changedFilesLog = changedFiles.join(',');
-    const ignoredFilesLog = filesToIgnore.join(',');
-    console.log(
-      `😰  Git shows some files have been changed. Changed files: ${changedFilesLog}. (ignoring ${ignoredFilesLog})`,
-    );
-
```

Comment:
Listing the actual files that have changed is probably useful to someone debugging their failed CI run. Any way we could keep this behaviour?

Reply:
It's still there, see check-no-changes.sh :)